# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gatling/libowfat.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gatling-0.8/gatling.c.vanilla	2007-01-15 13:13:01.000000000 +0000
+++ gatling-0.8/gatling.c	2007-01-15 13:13:35.000000000 +0000
@@ -865,6 +865,7 @@
   return (c!='"' && c!='%' && c>=' ' && c!='+');
 }
 
+#if 0
 unsigned long fmt_urlencoded(char* dest,const char* src,unsigned long len) {
   register const unsigned char* s=(const unsigned char*) src;
   unsigned long written=0,i;
@@ -882,6 +883,7 @@
   }
   return written;
 }
+#endif
 
 void catencoded(array* a,char* s) {
   unsigned int len=str_len(s);
